From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arnaldo Carvalho de Melo <acme@redhat.com>
Date: Tue, 27 Jan 2026 01:15:47 -0300
Subject: [PATCH 188/196] perf units: Constify variables storing the result of
 strchr() on const tables

As newer glibcs will propagate the const attribute of the searched table
to its return.

Signed-off-by: Arnaldo Carvalho de Melo <acme@redhat.com>
(cherry picked from commit 0e14cb3b24f8f301cf6490a4493afc98321ed5bb)

diff --git a/tools/perf/util/units.c b/tools/perf/util/units.c
index 4c6a86e1cb54b..0bbacf5a29aaf 100644
--- a/tools/perf/util/units.c
+++ b/tools/perf/util/units.c
@@ -12,7 +12,7 @@ unsigned long parse_tag_value(const char *str, struct parse_tag *tags)
 	struct parse_tag *i = tags;
 
 	while (i->tag) {
-		char *s = strchr(str, i->tag);
+		const char *s = strchr(str, i->tag);
 
 		if (s) {
 			unsigned long int value;
-- 
2.43.0

